files/zh-tw/web/html/element/article/index.md

```diff
@@ -0,0 +1,153 @@
+---
+title: "<article>: 文章內容元素"
+slug: Web/HTML/Element/article
+---
+
+{{HTMLSidebar}}
+
+**`<article>`** [HTML](/zh-TW/docs/Web/HTML) 元素代表文件、頁面、應用程式或站點中的一個獨立組成部分，該部分旨在獨立分發或重複使用（例如，在聯合編輯中）。例如：論壇帖子、雜誌或報紙文章、部落格文章、產品卡片、用戶提交的評論、互動小工具或小裝置，或任何其他獨立的內容項目。
+
+{{EmbedInteractiveExample("pages/tabbed/article.html", "tabbed-standard")}}
+
+一個文件可以包含多個文章；例如，在一個按讀者滾動顯示每篇文章文本的部落格上，每篇文章都可以包含在 `<article>` 元素中，可能包含一個或多個 `<section>`。
+
+<table class="properties">
+  <tbody>
+    <tr>
+      <th scope="row">
+        <a href="/zh-TW/docs/Web/HTML/Content_categories"
+          >內容類型</a
+        >
+      </th>
+      <td>
+        <a href="/zh-TW/docs/Web/HTML/Content_categories#flow_content"
+          >流動內容</a
+        >、
+        <a

```

Comment:
The line break will produce a "space" between the punctuation and Chinese words ([translation-guide](https://github.com/mdn/translated-content/blob/main/docs/zh-tw/translation-guide.md#%E4%B8%AD%E6%96%87%E5%8F%8A%E6%8B%89%E4%B8%81%E8%AA%9E%E7%B3%BB%E6%96%87%E5%AD%97%E9%96%93%E4%B8%8D%E5%8A%A0%E4%BF%AE%E9%A3%BE%E7%A9%BA%E7%99%BD)):

```suggestion
        >、<a
```